src/topology/bounded_continuous_function.lean

```diff
@@ -257,6 +260,7 @@ begin
 
   /- Associate to every function a discrete approximation, mapping each point in `tα`
   to a point in `tβ` close to its true image by the function. -/
+  letI := @coe_fn_trans ↥A _ _  (@bounded_continuous_function.has_coe_to_fun α β _ _),
```

Comment:
This is strange that you need this.

Reply:
This is discussed in https://leanprover.zulipchat.com/#narrow/stream/116395-maths/topic/uniform.20split Indeed the type class mechanism is behaving strangely. There is also a fix in `exponential.lean` that shouldn't be needed.